pkg/ride/compiler/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+//go:generate peg -output=compiler.peg.go compiler.peg
```

Comment:
Please add `README.md` or comments about what utility is used and what this binary does.